Boot2Docker.iss

```diff
@@ -17,7 +17,7 @@
 ; https://www.virtualbox.org/wiki/Downloads
 ; Then, run "VirtualBox-x.x.x-xxx-Win.exe --extract --path ."
 #define virtualBoxCommon ".\VirtualBox\common.cab"
-#define virtualBoxMsi ".\VirtualBox\VirtualBox-4.3.20-r96997-MultiArch_amd64.msi"
+#define virtualBoxMsi ".\VirtualBox\VirtualBox-4.3.26-r98988-MultiArch_amd64.msi"
```

Comment:
@tianon umm actually on build.sh side some exe extracts  msi file with version in it. but we can do `mv *_amd64.msi virtualbox_amd64.msi` and just use virtualbox_amd64.msi` here. How does that sound to you?